flixel/graphics/tile/FlxDrawQuadsItem.hx

```diff
@@ -128,7 +128,8 @@ class FlxDrawQuadsItem extends FlxDrawBaseItem<FlxDrawQuadsItem>
 
 		setParameterValue(shader.hasTransform, true);
 		setParameterValue(shader.hasColorTransform, colored || hasColorOffsets);
-
+		
+		camera.canvas.graphics.overrideBlendMode(blend);
```

Comment:
We need OpenFL version checks here to keep compatibility with OpenFL 8.0.0 - 8.7.0

Reply:
Remind me, how should that look?

Reply:
Done, I think.